index.html

```diff
@@ -82,6 +82,21 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="footer">   
+            <img class="some-icons" src="img/some-icons.svg" alt="facebook and instagram icons">
```

Comment:
Alt text should probably be in Swedish as the rest of the website is in Swedish? :)

Also, add `width` and `height` attributes to the [image tag](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/).